content/kubernetes/release-notes/k8s-6-2-18-41-jan-2023.md

```diff
@@ -17,18 +17,24 @@ New images and fixes are listed below. Refer to [6.2.18-41 (Dec 2022]({{<relref
 
 ## Images
 
+DockerHub images are available at `docker.io/`.
```

Comment:
I don't think we need to mention this. The image strings below (`redislabs/redis:6.2.18-72`) are automatically resolved to `docker.io/...` when needed. Deployment manifests typically use the hostless syntax to pull from Docker Hub.